packages/react/src/components/PieChartCard/PieChartCard.jsx

```diff
@@ -248,5 +253,5 @@ const PieChartCard = ({
 };
 
 PieChartCard.propTypes = propTypes;
-PieChartCard.defaultProps = defaultProps;
+PieChart.defaultProps = defaultProps;
```

Comment:
Was this a mistake? I noticed it b/c it removes a bunch of defaultProps from the publicAPI change.

Reply:
Definitely a mistake. I'll try and change today

Reply:
Fixed